src/backend/expungeservice/models/helpers/charge_creator.py

```diff
@@ -4,7 +4,7 @@
 
 from dacite import from_dict
 
-from expungeservice.models.charge_classifier import ChargeClassifier
+from expungeservice.models.helpers.charge_classifier import ChargeClassifier
```

Comment:
I'm not quite sure if I agree entirely with this name change, but that's not really important. What I think is important is that the `generator.py`, as far as I can tell, is only used for tests. Shouldn't the `generator.py` file be in the test directory?

Reply:
Feel free to suggest a different name. It is also used for the [mock backend](https://github.com/codeforpdx/recordexpungPDX/pull/628/files#diff-d84ab9e795dfeb073aa4f52331a51b6eR5).

Reply:
I'm honestly not familiar with the mock backend. Personally, I feel any code that does not contribute to the production app should be in its own directory, such as test, or whatever directory name is suitable. 

As far as the naming goes, when I see a directory named `helper` then I tend to think the files in the helper classes are to dry up code between different modules. This is a factory class, and it does help/facilitate the creation of a charge class. Although putting it in a directory with a class that is not used as part of the main app doesn't seem quite right.

Anyway, this is probably not worth talking about much. We can discuss more in person. Feel free to merge. I didn't see anything else that stood out on me. Thanks for the PR 💫